src/InteractiveMars/Mars.tsx

```diff
@@ -0,0 +1,48 @@
+/* TO-DO 
+- Make marker onClick zoom less janky (or disable alltogether)
+- Label markers with rover names without having to hover? (ThreeJS text geometry?)
+*/
+
+import React, {FunctionComponent} from "react";
+import ReactGlobe, {Marker} from "react-globe";
+import {markers} from "./Markers"
+import "tippy.js/dist/tippy.css";
+import "tippy.js/animations/scale.css";
+
+
+//Function to render onHover tooltips
+
+export function renderToolTipsOnHover(marker: Marker): string {
+  return `ROVER: ${marker.rover}`;
+}
```

Comment:
how come this is exported - isn't it only useful inside this component?

Reply:
oh - because of the test.

That's a bit of an anti-pattern. (/ a code smell)
ie - its normally bad practice to make something public / export something just because you want to test the function.

in general it implies that
- your component / class / module it too large and complex
- and therefore you can't easily test just the actually public methods
- so you end up making your actual code slightly less good (more exported things) just to satisfy tests.

which in general is the opposite of what you want to be doing.


So the ideal test here would instead

- load up the `Mars` Component.
- click one of the landing sites
- assert that you can now see an element that says `ROVER: Spirit` etc.

But that may or may not be really tricky in this case.
I'm kind of happy to let you get away with it this time and get it merged in though.

Maybe I'll revisit later! :)